src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -1212,9 +1212,13 @@ QString QgsPostgresConn::quotedValue( const QVariant &value )
   {
     case QVariant::Int:
     case QVariant::LongLong:
+    case QVariant::ULongLong:
```

Comment:
Hi, if I might add my two cents, PostgreSQL doesn't support unsigned integer types, so there is no need for ULongLong here.

This message from 2013 explains the reason: https://www.postgresql.org/message-id/CAEcSYX+Arn7y4FeYPp6ZgbiiiMfZYmsn9aUyotZB-MA1n5hTOw@mail.gmail.com

And it is still true as of 2020: https://www.postgresql.org/docs/12/datatype-numeric.html (look at the ranges); even for serial/bigserial pseudotypes, you can see that the maximum value is the same as the upper value for signed integers/big integers.

Even for date and time types, they are implemented as singed values: https://tapoueh.org/blog/2018/04/postgresql-data-types-date-timestamp-and-time-zones/

So, no need for accounting for unsigned values here and in line 1627 below.

Reply:
You are right, I will change it !